CMakeLists.txt

```diff
@@ -461,19 +461,14 @@ if(CMAKE_CXX_COMPILER_ID MATCHES Clang)
 elseif(CMAKE_CXX_COMPILER_ID MATCHES GNU)
 	add_compile_options(
 		-Wall
-		-Wc++11-compat
 		-Wextra
-		-Wno-format-zero-length
-		-Wparentheses
```

Comment:
why disable `Wparentheses`?

Reply:
it is not disabled, it is already covered by `-Wall`.